client.go

```diff
@@ -66,6 +66,16 @@ func (c *Client) Post(resource string, data url.Values) ([]byte, error) {
 	return c.do(req)
 }
 
+func (c *Client) Put(resource string, data url.Values) ([]byte, error) {
+	req, err := http.NewRequest("PUT", c.endpoint+resource, strings.NewReader(data.Encode()))
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
btw: the `AddCard()` would be also a good addition, I like the validation funcs etc. -- what would you think of

```go
card, err := list.AddCard(trello.Card{
 // ...
 // ...
})

// and/or
card, err := board.AddCard(trello.Card{
 // ...
 // ...
})
```